src/express/outgoing/model.ts

```diff
@@ -6,39 +6,43 @@ import { IOutgoingFile } from './interface';
 const outgoingFileSchema = new mongoose.Schema<IOutgoingFile & mongoose.Document>(
     {
         approvers: {
-            objectId: { type: 'ObjectId', required: true },
+            type: [String],
+            required: true,
         },
-        name: {
+        fileName: {
```

Comment:
lets change it to name
```suggestion
        name: {
```

Reply:
done